src/service/performance-impl.js

```diff
@@ -221,7 +221,7 @@ export class Performance {
    * @private
    */
   setFlushParams_(params) {
-    this.viewer_.setFlushParams(params);
+    this.viewer_.sendMessageCancelUnsent('setFlushParams', params, false);
```

Comment:
I'm going to create another PR for the refactoring of `awaitResponse`.
I'm not sure whether the viewer is respecting the `true/false` value set by us or just having predefined behavior depending on the `eventType` (postMessage for certain type of event without checking the `awaitResponse` param)